README.rst

```diff
@@ -32,61 +32,27 @@ You can use Vint with
 Configure
 ---------
 
-Vint will read config files on the following priority order:
+Vint will read config files from files named ``.vintrc.yaml``, ``.vintrc.yml``,
+or ``.vintrc``.
 
--  `User config <#user-config>`__:
--  e.g. ``~/.vintrc.yaml`` (the filename can be ``.vintrc.yml`` or ``.vintrc``)
+1. any of these files is read from your home directory.
 
--  `Project config <#project-config>`__:
--  e.g. ``path/to/proj/.vintrc.yaml`` (the filename can be ``.vintrc.yml`` or ``.vintrc``)
+2. any of these files is read based on the current directory, which allows for
+   project specific settings.
```

Comment:
be more specific here with regard to upward search, and where it starts (check the source)

Reply:
https://github.com/Kuniwak/vint/issues/311